src/manifest/owned/pfm.rs

```diff
@@ -114,10 +136,25 @@ pub struct Rw {
             serialize_with = "crate::serde::se_bin",
         )
     )]
-    pub flags: u8,
+    flags: u8,
     pub region: Region,
 }
 
+impl Rw {
+    /// Gets the `RwFailurePolicy` flag value.
+    pub fn get_rw_failure_policy(&self) -> RwFailurePolicy {
```

Comment:
`s/get_//`; in general, Rust getters don't begin with `get`.

Reply:
done!